app/models/viewmodels/notificationspage/converters/ControlResponseConverter.scala

```diff
@@ -54,27 +54,17 @@ class ControlResponseConverter @Inject()(decoder: Decoder, dateTimeFormatter: Da
     }
 
   private def buildActionCodeExplanation(actionCode: String)(implicit messages: Messages): Option[String] =
-    decoder.actionCode(actionCode).map(code => paragraph(messages(code.contentKey)))
-
-  private def buildErrorExplanation(errorCode: String)(implicit messages: Messages): Option[String] = {
-    val isChiefError = decoder.chiefErrorCode(errorCode).isDefined
-
-    if (isChiefError) buildCHIEFErrorExplanation(errorCode)
-    else buildILEErrorExplanation(errorCode)
-  }
+    decoder.actionCode(actionCode).map(code => paragraph(messages(code.messageKey)))
 
   // TODO move logging for missing error codes to backend
-  private def buildILEErrorExplanation(errorCode: String)(implicit messages: Messages): Option[String] =
-    decoder.ileErrorCode(errorCode).map(error => paragraph(messages(error.contentKey))) match {
+  private def buildErrorExplanation(errorCode: String)(implicit messages: Messages): Option[String] =
+    decoder.error(errorCode).map(error => paragraph(messages(error.messageKey))) match {
       case None =>
         logger.info(s"Received inventoryLinkingControlResponse with unknown error code: $errorCode")
         None
```

Comment:
Maybe return just the "raw" error code if it can't be "translated"?   Or a canned message like "Unknown error XXX" .   Then the user knows there was an error and can report it  (and someone can investigate and possibly update the csv file).

In which case, doesn't need to return Option.

Reply:
This was actually the requirement from the business. Not to display anything for unknown errors. We are only logging this, which by the way is going to be moved to the BE. And then the logging level possibly changed to `warn`.